homeassistant/components/upb/__init__.py

```diff
@@ -65,6 +82,17 @@ async def async_unload_entry(hass: HomeAssistant, config_entry: ConfigEntry) ->
     return unload_ok
 
 
+async def _async_update_listener(hass: HomeAssistant, config_entry: ConfigEntry):
+    """Handle options update."""
+    upb = hass.data[DOMAIN][config_entry.entry_id]["upb"]
+    tx_count = config_entry.options.get(CONF_TX_COUNT, None)
+    if tx_count:
```

Comment:
Can use `:=` here.
`None` not required on `get`.

Reply:
Done